src/EventSubscriber/DeepLinkSubscriber.php

```diff
@@ -36,18 +36,18 @@ class DeepLinkSubscriber implements EventSubscriberInterface
     private $responseFactory;
 
     /**
-     * @var Contentful
+     * @var UrlGeneratorInterface
      */
-    private $contentful;
+    private $urlGenerator;
 
     /**
-     * @param ResponseFactory $responseFactory
-     * @param Contentful      $contentful
+     * @param ResponseFactory       $responseFactory
+     * @param UrlGeneratorInterface $urlGenerator
      */
-    public function __construct(ResponseFactory $responseFactory, Contentful $contentful)
+    public function __construct(ResponseFactory $responseFactory, UrlGeneratorInterface $urlGenerator)
```

Comment:
The `Contentful` object was a leftover from a previous version, and wasn't needed any more